cli/src/cli_util.rs

```diff
@@ -1595,6 +1595,38 @@ Then run `jj squash` to move the resolution into the conflicted commit."#,
     }
 }
 
+pub fn choose_commit<'a>(
+    ui: &mut Ui,
+    workspace_command: &WorkspaceCommandHelper,
```

Comment:
nit: might be better to pass `template: &` instead to minimize dependency on `&WorkspaceCommandHelper`